lms/services/lti_grading/_v13.py

```diff
@@ -47,6 +47,8 @@ def read_result(self, grading_id) -> GradingResult:
             # There's currently a bug in Blackboard's LTIA implementation.
             # Using the user filter in the request removes the comment field in the response's body.
             del params["user_id"]
+            # We add an explicit limit parameter to fetch all results
+            params["limit"] = 1000
```

Comment:
This is basically the bugfix, at least for now. We might hit the page limit again but we'll get a sentry notification about it.

We use the same 1000 limit for example in Canvas API calls.